CMakeLists.txt

```diff
@@ -185,6 +185,13 @@ ecbuild_add_option( FEATURE ETRANS
                     DEFAULT OFF
                     DESCRIPTION "Include Limited-Area-Model Transforms" )
 
+	   
+ecbuild_add_option( FEATURE ECTRANS4PY
+                    DEFAULT OFF
+		    CONDITION HAVE_ETRANS
```

Comment:
If I understand right, `HAVE_DOUBLE_PRECISION` is effectively a condition for `ECTRANS4PY`. Why not make it "official" and put it as a condition here?